src/test/java/core/basesyntax/TestObjects.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestObjects {
+    public static final String ORANGE = "Orange";
+    public static final String APPLE = "Apple";
+    public static final String BANANA = "Banana";
+    public static final Integer FRUIT_QUANTITY = 100;
+    public static final String CORRECT_PATH = "src/test/resources/Input.csv";
+    public static final String INCORRECT_PATH = "InputNew.csv";
+    public static final String INVALID_PATH = "/invalid/path/test.csv";
+    public static final String FIRST_LINE = "type,fruit,quantity";
+    public static final String SECOND_LINE = "b,banana,20";
+    public static final String THIRD_LINE = "b,apple,100";
+    public static final String HEADER = "fruit,quantity";
+    public static final int HEADER_NUMBER = 0;
+    public static final String SPLITTER = ",";
+    public static final List<String> LINES_FROM_INPUT_FILE = new ArrayList<>();
```

Comment:
I suggest to remove this constant and create a separate object where it is required. I could cause problems when 2 test running simultaneously